data/json/body_parts.json

```diff
@@ -68,8 +68,7 @@
   {
     "id": "arm_l",
     "type": "body_part",
-    "name": "left arm",
-    "name_plural": "arms",
```

Comment:
This wasn't changed to the new format, because it doesn't actually use the new format: "name_plural" here means the name for a pair of body parts, and was only named "name_plural" because it was (incorrectly) implemented as a plural name. If this is to be changed, the proper change would be renaming it from "name_plural" to something like "name_pair".

Reply:
Thanks for clearing that up. I have no issue with it remaining as it is, but we might run into the same problem when someone else is looking for name_plural in the files and tries to change it.

Reply:
We would probably like more than two body parts maximum eventually, so using name_plural could be okay.

Reply:
It prevents proper translation of the plural name in languages without plurals (such as Chinese).

Reply:
Hmm... What about other plural strings (like item names)? Are they not translatable to these languages too?

Reply:
Item names are fine becasue the plural string is the exact plural form of the singlular string. But here it's not the case: the exact plural form of "left arm" is "left arms", and the exact singular form of "arms" is "arm". If we translate "left arm" and "arms" to a single string in languages without plural forms, the result is that we either translate "arms" to "left arms", or translate "left arm" to "arm", which do not make sense in many contexts such as the encumberance menu and amor sorting menu.

To clarify it let me give an example in Chinese:
If we translate the two strings separately:
```
left arm -> 左(left)臂(arm)
arms     -> 手臂(arms)
```
But if we treat them as singular and plural forms:
```
left arm / arms -> 左(left)臂(arm(s))
```
or
```
left arm / arms -> 手臂(arm(s))
```

That said, if gettext and Transifex has the functionality to override the plural settings of a language for a specific string, then we can treat these strings as singular and plural forms. Otherwise, we need to specify them separately.